### PR TITLE
Prep v5.4.0 release

### PR DESCRIPTION
**Pull request by schisamo**
This will allow us to update the default version of Omnibus installed by the `artifactory-pro` cookbook (used by the `artifactory_omnibus_publisher` resource).

/cc @chef/omnibus-maintainers @chef/engineering-services

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
 Omnibus CHANGELOG
 =================
 
+v5.4.0 (April 18, 2016)
+-----------------------
+### New Features
+- Include license and version manifest in generated `*.metadata.json` (#656)
+- Deprecate the `- -version-manifest` on `omnibus publish` (#656)
```
Comment (stevendanna) [inline CHANGELOG.md]: "The `--version-manifest` option on"

**Comment by sersut** [comment]
👍

**Comment by scotthain** [comment]
👍

**Comment by patrick-wright** [comment]
👍

**Pull request merged**